### PR TITLE
Feature/add activity

### PR DESCRIPTION
**Pull request by magnify**

### CODE REVIEW

File: web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss]: magin?

File: web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/radio.twig
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/radio.twig]: Wierd space?

File: web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig:5]: Perhaps default should be en empty icon?

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig:619]: Buttons in pattern-lab?

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig]: {% if form %}

**Review by tuj — CHANGES_REQUESTED** [review]

File: web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss
Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss]: Whoa!

Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss]: magin is the new margin.

File: web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig
Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig:5]: Let's stick with this for now. We will probably get back to it.

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig
Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig:619]: Waiting for merge with 5 step form code, after merge I will take a look at getting it to work in Pattern Lab.

**Comment by magnify** [comment]
Commit changes, please review.

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig
Comment (tuj) [reply web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig:619]: Ok

**Pull request merged**
